Makefile

```diff
@@ -255,11 +255,10 @@ ifeq ($(CI), true)
 endif
 	oc apply -f testing/costmanagement-metrics-cfg_v1beta1_costmanagementmetricsconfig.yaml
 
-SECRET_NAME = $(shell oc get secrets -o name | grep -m 1 koku-metrics-controller-manager-token-)
 .PHONY: get-token-and-cert
-get-token-and-cert:  ## Get a token from a running K8s cluster for local development.
```

Comment:
The comment here is what populates `make help`:
```
$ make help

Usage:
  make <target>

General
  help             Display this help.

...

Deployment
...
  get-token-and-cert  Get a token from a running K8s cluster for local development.

```

Reply:
Oh I see. Thanks for the catch! I've added a comment.